tests/test_discovery_sample/tests_sample.py

```diff
@@ -29,6 +30,18 @@ class EmptyTestCase(TestCase):
     pass
 
 
+class TaggedTestCase(TestCase):
+    tags = ('slow',)
```

Comment:
Any reason not to use `tags` decorator for the test class? Setting `tags` directly is not documented.